### ISSUE TITLE
Powershell stager for http(s) listener

### ISSUE THREAD

**Issue by zinzloun** [EmpireProject/Empire#1309]
## Empire Version
2.5

## OS Information (Linux flavor, Python version)
kali-rolling 2019.1 x64, py 2.7.15+

## Expected behavior and description of the error, including any actions taken immediately prior to the error. The more detail the better.
The powershell stagers script created for the http listener, having https enabled, when executed generates an error in the communication channel with the server. The problem is the TLS protocol version used by default in powershell cli which is 1.0
Tested on 
- win10pro x64, PS version 5.1
- win 7pro SP1 x64, PS version 3.0

## Screenshot of error, embedded text output, or Pastebin link to the error
My http listener configuration
![http](https://user-images.githubusercontent.com/24391516/51258742-f78f2900-19aa-11e9-9f0b-45c7c9322875.PNG)

My stager multi/launcher configuration, also tested with windows/launcher_bat and windows/macro
![stager](https://user-images.githubusercontent.com/24391516/51259111-c6fbbf00-19ab-11e9-8dc6-cb4185e2a8b3.PNG)

PS payload executed on Win10 ["Request rejected: It's not possible to create a secure SSL\TLS channel..."]
![ps error win 10](https://user-images.githubusercontent.com/24391516/51263290-4097ab00-19b4-11e9-9b2e-228f396ed113.png)

Same thing on Win7
![ps tls err win72](https://user-images.githubusercontent.com/24391516/51263319-52794e00-19b4-11e9-8017-8d01edb989dd.png)

Packets exchanged between the server 192.168.1.66 and the win10 machine, same thing happens with the win7 machine
![wireshark tls err](https://user-images.githubusercontent.com/24391516/51260909-93229880-19af-11e9-8461-74da955ecb3b.PNG)

TLS version of the generated certificate used by the listener
![listener_cert](https://user-images.githubusercontent.com/24391516/51261213-22c84700-19b0-11e9-8ea6-73f5acfcc3f3.PNG)

## Any additional information
I solved the problem forcing the payload agent generation on **http.py** file to use **TLS1.2**
![diff](https://user-images.githubusercontent.com/24391516/51261811-48a21b80-19b1-11e9-9a59-90800b77b72a.PNG)

I don't know if it's the best approach or solution, in my case it solves all the problems with the stagers
![empire agents](https://user-images.githubusercontent.com/24391516/51262078-e5fd4f80-19b1-11e9-8fe5-b48bd2ff0ca8.PNG)

Any additional comments will be very appreciated.
Thank you to the Empire team for this wonderful framework

**Comment by kylesmithit** [comment]
I this problem and @zinzloun 's solution resolved it for me as well.

**Comment by mr64bit** [comment]
What Win10 build and .NET version are you running on? I haven't been able to reproduce this.

**Comment by zinzloun** [comment]
> 
> 
> What Win10 build and .NET version are you running on? I haven't been able to reproduce this.

I do not think that the problem is related to the .Net, as far as I know the .Net set the TLS version available but not the default used in PS. Anyway following the information that you have requested:
![cattura](https://user-images.githubusercontent.com/24391516/53586886-6a361b80-3b89-11e9-872f-7c115f41d8da.PNG)
![cattura2](https://user-images.githubusercontent.com/24391516/53586955-8c2f9e00-3b89-11e9-85b0-bc82ab904d04.PNG)

This is the CLR version used in my PS enviroment:
![cattura3](https://user-images.githubusercontent.com/24391516/53587631-2cd28d80-3b8b-11e9-992d-5ade4d60a072.PNG)

**Comment by bgospodaru** [comment]
Can confirm. The same issue happened to me as well and @zinzloun's solution worked.
